docs/examples/.env.truenas.example

```
@@ -0,0 +1,64 @@
+SMTP_SSL=false
+EMAIL_FROM_ADDRESS=noreply@falloutshelter.com
+EMAIL_FROM_NAME=Fallout Shelter
+
+# ============================================
+# RUSTFS (external S3-compatible storage)
+# Internal API endpoint for boto3 client connections
+# ============================================
+RUSTFS_HOSTNAME=s3-api.evillab.dev
+RUSTFS_PORT=443
+RUSTFS_USE_HTTPS=True
+RUSTFS_ACCESS_KEY=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RUSTFS_ACCESS_KEY key should go before the RUSTFS_HOSTNAME key

(UnorderedKey)